packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -267,9 +269,30 @@ Element.propTypes = {
  * @return {null|*} Element or null if does not map to video/image.
  */
 function MediaElement(props) {
-  const { isTranscoding, isMuting, isTrimming } = props.resource;
+  const {
+    isResourceTrimmingById,
+    isResourceMutingById,
+    isResourceTranscodingById,
+  } = useLocalMedia(
+    ({
+      state: {
+        isResourceMutingById,
+        isResourceTrimmingById,
+        isResourceTranscodingById,
+      },
+    }) => ({
+      isResourceMutingById,
+      isResourceTrimmingById,
+      isResourceTranscodingById,
+    })
+  );
+  const { id } = props.resource;
 
-  if (isTranscoding || isMuting || isTrimming) {
+  if (
+    isResourceTrimmingById(id) ||
```

Comment:
Q: Wouldn't `isResourceProcessing` already check for all these? Although, I'm also seeing `MUTED` in the list of the statuses for `isResourceProcessing`. I probably don't have enough context here right away, but is media still processing if it's already `muted`? I thought it would only be processing if it's any of the statuses with `-ING` ending.

Reply:
It is still processing, as after is muted, it still needs to be uploaded.

Reply:
Okay, but wouldn't the status then be `UPLOADING`?

Reply:
In this case we don't want tooltip to display.

Reply:
I'm just generally trying to understand why are `MUTED`, `TRIMMED` etc. statuses necessary. If after `TRIMMING` we need to upload and change to `UPLOADING`, why do we still need separately `MUTED`, `TRIMMED`, etc.?

Reply:
> why do we still need separately `MUTED`, `TRIMMED`, etc.?

That is a great question. You are right, we probably don't need `MUTED`, `TRIMMED` etc., since we immediately go over to `UPLOADING` after that by calling `startUploading`:

https://github.com/google/web-stories-wp/blob/4a614ce56910ca9efb00bcd1eb49c4406a37b456/packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js#L316

Reducers like `finishMuting` just update `isMuted` on the current resource, which is what we mostly care about. So theoretically we would not need to update `state` here:

https://github.com/google/web-stories-wp/blob/4a614ce56910ca9efb00bcd1eb49c4406a37b456/packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js#L261-L272

Reply:
Alright, I guess at least it's not necessary to include those statuses into `isResourceProcessing` and `isResourceProcessingById` (per current naming)? Since these states will be changed right away, hence not really used + during that tiny moment when those states are assigned, nothing is processing.
It would make the code a bit more readable / understandable as well.

Reply:
So should I removed muted, trimmed and transcoded states then?

Reply:
Yes; would be great if you could give that a shot.

Reply:
5c375b2038251c049903fd16208ffcd45f0444aa is not enough

My understanding is that we want to remove lines like this as well:

https://github.com/google/web-stories-wp/blob/23092d2987419a6c61c8c52e794cd08b79f7d3dd/packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js#L267

And of course remove `MUTED` from `ITEM_STATUS`.

Completely getting rid of `ITEM_STATUS.MUTED` et al.

Reply:
Once a mute / transcode / trim is complete, should the state return to pending?

Reply:
No, `PENDING` means a file pending any processing/upload whatsoever; a file at the very end of the queue.

We don't need to change the state at all. The item can stay in its current state as it will be changed to `UPLOADING` right away on this line:

https://github.com/google/web-stories-wp/blob/4a614ce56910ca9efb00bcd1eb49c4406a37b456/packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js#L316

Reply:
We do needed transcoded state, as it used in current implementation. I think we should leave this for now and remove this in a follow up PR.

Reply:
@spacedmonkey And where is it used? But then you'd need to undo 5c375b2038251c049903fd16208ffcd45f0444aa, no? 🤔

Reply:
@swissspidy trancoded state is used here https://github.com/google/web-stories-wp/blob/91d7271af6a7f17b42e0be4bbaef5332fecd7170/packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js#L89-L95

The relays on transcode to be complete before regenerating the poster image. 

I do agree that the trancoded, muted and trimmed states seem a little pointless, however, this PR is already massive. Can we please remove this in another PR?